CRM/Contact/Form/Search.php

```diff
@@ -916,6 +916,7 @@ protected function loadMetadata() {
     // can't by pass acls by passing search criteria in the url.
```

Comment:
@eileenmcnaughton just saying this feels like its going to get really annoying doing this

Reply:
@seamuslee001 there is a pretty good case for NOT doing anything more to add features like this to the advanced search. However, NYSS has invested in trying to improve this area & we've done a bunch of cleanup so we should at least get a good working pattern. WMF is looking to sponsor discovery on a search replacement